en/docs/setup/migration-guide.md

```diff
@@ -429,9 +429,9 @@ Now let's see how to do the blue-green deployment with WSO2 Identity Server.
 
 1.  Create new databases for the new WSO2 Identity Server version (5.9.0) 
     that you are migrating to.
-2.  Unzip a WSO2 Identity Server 5.9.0 distribution (use a WUM updated distribution 
+2.  Unzip a WSO2 Identity Server 5.9.0 (use a WUM updated distribution 
```

Comment:
if we removing the word distribution here, we should also remove "a" from "Unzip **a** WSO2 Identity Server 5.9.0"